### ISSUE TITLE
v2.4.2 browser bundle does not properly export precompileString (and others)

### ISSUE THREAD

**Issue by metatoaster** [mozilla/nunjucks#785]
Code to reproduce this (stick the `nunjucks.js` in `/browser` with the following)

```
<html><head><title>Nunjucks Test</title>
<script type="text/javascript" src="nunjucks.js"></script>
</head>
<script>
var hello = nunjucks.renderString('Hello {{ user }}', {user: 'World' });
var tmpl = nunjucks.precompileString('Hello {{ user }}');
</script>
</body></html>
```

`renderString` behaves as intended, while `precompileString` (and a couple others like `precompile`) simply reported as `is not a function` error as they are not properly defined (even though they appear in the autocomplete from within a browser).  Is the [documentation](http://mozilla.github.io/nunjucks/api.html#precompile) related to precompiling accurate or the browser bundle being incomplete?

**Comment by vecmezoni** [comment]
Precompile is stripped from the browser bundle https://github.com/mozilla/nunjucks/blob/master/bin/bundle.js#L45.

Is there any use case for precompiling templates in browser?

**Comment by metatoaster** [comment]
There are, however I have more or less redesigned my thing to not rely on the browser doing anything once I figured out how this library works, _however_, some JavaScript/templating/ui libraries/frameworks (such as [Plone 5's through-the-web JavaScript bundling framework](https://plone-training.readthedocs.io/en/latest/javascript/javascript-3-ttw.html)) provide means to rely on user's browsers to do transpilations/minifying artifact bundles which then gets sent back to the server for serving (say after a template change and what not).

Another use-case is giving the users ability to generate templates locally in their browser, like inside their webstorage or whatever (though this is a stupid idea, but Plone 5's TTW bundling/artifact generation is something I am considering doing with nunjucks, but given that I have accustom myself with the library I can probably figure out a way to generate a bundle with that).

At the very least, it definitely would be great to document the lack of the precompiling template functionality in the web bundle somewhere to save future users some minutes/hours in wondering why their code doesn't work within a browser's developer window.

**Comment by vecmezoni** [comment]
Ok, for now i'd suggest to fix the documentation.

**Comment by fdintino** [comment]
This will be incidentally fixed with the merge of #1053

**Comment by fdintino** [comment]
Fixed with merge of #1053, in release v3.1.0
